KeenClient.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name         = 'KeenClient'
-  spec.version      = '3.3.1'
+  spec.version      = '3.4'
```

Comment:
We've used the x.x.x convention for releases so far, so let's tweak this to be 3.4.0 instead of 3.4.